app/facades/foods_facade.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsFacade
+  def self.get_list(q)
```

Comment:
Here we are making two calls to the api, one to get the food list and another to get a count. This is where adding that param for `:totalHits` comes into play.
So here is an example of how you might be able to make one call and get both bits:
```
def self.foods_by_keyword(keyword)
    foods_results = FoodService.foods_by_keyword(keyword)
    x = foods_results[:foods].map do |food_result|
      Food.new(food_result)
    end
    total_hits, foods = [foods_results[:totalHits], x]
  end
end
```
the `total_hits, foods = [food_results[:totalHits], x]` is creating an array with our keyword search and total hits. Which we can call in the controller. See controller note